### ISSUE TITLE
Training form the rasa-x UI fails with TrainingError

### ISSUE THREAD

**Issue by ddombrowsky** [RasaHQ/rasa#3735]
**Rasa version**:

rasa --version
Rasa 1.0.0

**Rasa X version** (if used & relevant):

rasa/rasa-x:stable (docker)

**Python version**:

python3 --version 
Python 3.6.8

**Operating system** (windows, osx, ...):
Linux 143c6802c949 4.4.0-1083-aws #93-Ubuntu SMP Wed May 8 16:08:41 UTC 2019 x86_64 GNU/Linux

**Issue**:

The training button does nothing.  Error in the logs is:
```
rasa-x_1           | Model training failed with status '500'  and message '{"version":"1.0.0","status":"failure","message":"An unexpected error occurred during training. Error: expected str, bytes or os.PathLike object, not NoneType","reason":"TrainingError","details":{},"help":null,"code":500}'.
```
No idea what path it's looking for.
I followed the server deploy instructions at http://rasa.com/docs/rasa-x/deploy/

After fixing the issue in https://github.com/RasaHQ/rasa/issues/3726, I was able to talk to existing models as trained in my local instance.  Now, in the public deployment, I cannot train.

For that matter, none of the training data is present (no stories, no NLU data), and I have no idea how to import it into the postgres database.

**Comment by ddombrowsky** [comment]
Any clues on how to track this one down?

**Comment by akelad** [comment]
Thanks for raising this issue, @ricwo  will get back to you about it soon.

**Comment by ddombrowsky** [comment]
Maybe I'm misunderstanding something fundamental.  What about these points is not true?
* models trained using the "local" version can be used in the server version.
* there is on `domain.yml` or `domain.md` file on the server.  It's in the postgres database and there is no way to edit it through the UI.  This means new intents need to be added manually.
* These intents are not automatically imported with the model.
* The training will fail if the intents aren't present.  

Are all these things true?

**Comment by amn41** [comment]
hi @ddombrowsky - there's a new release of rasa x that brings in first-class support for the domain file. It'll be ready very soon, will post back here when it is.

**Comment by tmbo** [comment]
This is released and part of the 0.19.1 release. Please let us know if there are still issues with your workflow.
